Charles Boegner: Bootstrap-Project-I

MichaelBoegner opened the pull request:
Foundational Less files set up. Beggining outline of layout on scratch paper

MichaelBoegner committed:
Charles Boegner: Bootstrap-Project-I

Foundational Less files set up. Beggining outline of layout on scratch paper

dk574 commented:
Hey Charles. You're probably in the zone now with your project. :) Try to add more commits if you've made any changes. Other than that, keep hacking away. 👍

MichaelBoegner committed:
Charles Boegner: Bootstrap Project I

Resume outlined. Components researched. Implementing Jumbotron, List Group, Buttons, and Card. Components 3 out of 4 implemented. Bug with selfie.

MichaelBoegner committed:
Charles Boegner: Bootstrap-ProjectI

All four components added and objectives completed. Polishing.

dk574 commented:
Just saw your preview. I am very pleased with the progress so far! And the color scheme is bomb! 👍

MichaelBoegner committed:
Charles Boegner: Bootstrap Project I

Introduced company logo and completed entering data into experience table. Attempting to introduce media queries to account for mobile column needs

MichaelBoegner committed:
Charles Boegner: Bootstrap-Project-I

Introduced responsive company logo and accomodated mobile breakpoint via bootstrap col syntax. All Tasks completed as follows: 1. Responsive layout folows shown grid system. 2. 4 of the components listed were used. 3. A responsive image of myself was used. 4. A table was introduced that was relevant to the resume.

MichaelBoegner committed:
Charles Boegner: Bootstrap-Project-I

Titles introduced and styling for them unified. Resume completed.
